pytest_django/fixtures.py

```diff
@@ -46,7 +46,7 @@ def django_db_modify_db_settings_tox_suffix():
 def django_db_modify_db_settings_xdist_suffix(request):
     skip_if_no_django()
 
-    xdist_suffix = getattr(request.config, "slaveinput", {}).get("slaveid")
+    xdist_suffix = getattr(request.config, "workerinput", {}).get("workerid")
```

Comment:
I haven't followed the slave/worker work, but is it safe to just switch, or should we keep compatibility with the old names for a while? I.e. like this:

```suggestion
    workerinput = getattr(request.config, "workerinput", getattr(request.config, "slaveinput", {}))
    xdist_suffix = workerinput.get("workerid", workerinput.get("slaveid"))
```

Reply:
pytest-cov switched wholesale, so I think pytest-django should too

Reply:
Right, looks like it should be fine from the graph provided in https://github.com/pytest-dev/pytest-cov/pull/412#discussion_r440068695.